lib/python/rose/config_editor/__init__.py

```diff
@@ -550,6 +551,8 @@
 DIALOG_TITLE_IGNORE = "Ignore section"
 DIALOG_TITLE_INFO = "Information"
 DIALOG_TITLE_OPEN = "Open configuration"
+DIALOG_TITLE_LOAD_METADATA = "Load metadata"
+DIALOG_TITLE_MANAGE_METADATA = "Manage metadata"
```

Comment:
Just had another look at this...

The menu item says *Custom Metadata*, but the dialog says *Manage metadata*. I think we should use the same string for both, and what about *Manage Metadata Search Path* (or just *Metadata Search Path*). It is sort of what the `--meta-path` option help string says.

In addition, the menu item should display a "..." suffix since it opens up a dialog box.